projects/novo-elements/src/elements/form/FieldInteractionApi.ts

```diff
@@ -533,7 +533,14 @@ export class FieldInteractionApi {
 
   public modifyPickerConfig(
     key: string,
-    config: { format?: string; optionsUrl?: string; optionsUrlBuilder?: Function; optionsPromise?: any; options?: any[] },
+    config: {
+      format?: string;
+      optionsUrl?: string;
+      optionsUrlBuilder?: Function;
+      optionsPromise?: any;
+      options?: any[];
+      entityPicker?: boolean;
+    },
```

Comment:
instead of making this a boolean for entityPicker - how about making it a property of type "ResultsTemplateType" which can allow options like "multi-entity-picker" or "entity-picker"

Reply:
updated!